tonic_validate/validate_scorer.py

```diff
@@ -132,13 +137,14 @@ async def _score_item_rundata(
                     )
             benchmark_item = response.benchmark_item
             return RunData(
-                scores,
-                benchmark_item.question,
-                benchmark_item.answer,
-                response.llm_answer,
-                response.llm_context_list,
+                scores=scores,
+                reference_question=benchmark_item.question,
```

Comment:
We need this btw so the type errors if someone inputs something wrong isn't confusing